src/test/java/integration/controller/AuthControllerIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package integration.controller;
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.all_tables_apiApplication;
+import org.kainos.ea.all_tables_apiConfiguration;
+import org.kainos.ea.model.Login;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthControllerIntegrationTest {
+        String email = System.getenv("LOGIN_EMAIL");
+        String password = System.getenv("LOGIN_PASSWORD");
+    Login invalidLogin = new Login(
+            "admin",
+            "admin"
+    );
+
+    Login validLogin = new Login(
+            email,
+            password
+    );
+
+    static final DropwizardAppExtension<all_tables_apiConfiguration> APP = new DropwizardAppExtension<>(
+            all_tables_apiApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    @Test
+    void login_shouldReturn200_whenSuccessfulLogin() {
+        Response responseLogin = APP.client().target("http://localhost:8080/api/login")
+                .request()
+                .post(Entity.entity(validLogin, MediaType.APPLICATION_JSON_TYPE));
+
+        String responseToken = APP.client().target("http://localhost:8080/api/login")
+                .request()
+                .post(Entity.entity(validLogin, MediaType.APPLICATION_JSON_TYPE), String.class);
```

Comment:
you don't need to make another call you can do something like:

```suggestion
        String responseToken = responseLogin.entity(String.class);
```